app/va/va_profile/va_profile_client.py

```diff
@@ -51,12 +56,30 @@ def get_email(self, va_profile_id):
         )
         response = self._make_request(url, va_profile_id, self.EMAIL_BIO_TYPE)
 
-        most_recently_created_bio = self._get_most_recently_created_email_bio(response, va_profile_id)
-        email = most_recently_created_bio['emailAddressText']
-        self.statsd_client.incr("clients.va-profile.get-email.success")
-        return email
+        try:
+            sorted_bios = sorted(
+                response['bios'],
+                key=lambda bio: iso8601.parse_date(bio['createDate']),
+                reverse=True
+            )
+            if sorted_bios:
+                if sorted_bios[0].get("emailAddressText"):
+                    # The e-mail address attribute is present and not the empty string.
+                    self.statsd_client.incr("clients.va-profile.get-email.success")
```

Comment:
This metric is useless without a corresponding failure or total stat. Please add 
`self.statsd_client.incr("clients.va-profile.get-email.failure")` to just above the `self._raise_no_contact_info_exception`